jquery.flot.navigate.js

```diff
@@ -215,7 +215,8 @@ Licensed under the MIT License ~ http://threedubmedia.googlecode.com/files/MIT-L
             cursor: "move",
             frameRate: 60
         }
-    };
+    },
+    panAxes = undefined;
```

Comment:
I think this variable must be moved in the init(plot) function. Here it's shared by all plots but the init(plot) is called for every plot so a different panAxes will be allocated. Chances to have more plots using the same panAxes are really low but you may never know.